force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,9 +29,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
-    }
-
+       if (x!=null && y!=null && x>y)
+        {return 'Hello World!';} 
```

Comment:
Try to use some of the best practice for code formatting. https://www.salesforceben.com/salesforce-code-formatting-best-practices/ The formatting you have here is really hard to read, particularly with the longer control statements.